src/main/inAppNotifications/InAppNotificationsDialog.tsx

```diff
@@ -0,0 +1,21 @@
+import { DialogContent } from '@mui/material';
+import NotificationsNoneOutlinedIcon from '@mui/icons-material/NotificationsNoneOutlined';
+import { useTranslation } from 'react-i18next';
+import DialogWithGrid from '../../core/ui/dialog/DialogWithGrid';
+import DialogHeader from '../../core/ui/dialog/DialogHeader';
+import { useInAppNotificationsContext } from './InAppNotificationsContext';
+import { InAppNotificationsList } from './InAppNotificationsList';
+
+export const InAppNotificationsDialog = () => {
+  const { t } = useTranslation();
+  const { isOpen, setIsOpen } = useInAppNotificationsContext();
+
+  return (
```

Comment:
We are not doing it often, but I think in this case that this component sits very high and will be rendered in every page, it would be good if we just return `null` if is not open to avoid rendering unnecessary things. 
This dialog doesn't have any state that we want to keep when closed and I guess that was your intention already with that `{isOpen && <InAppNotificationsList />}` to avoid calling the query?

Reply:
Yes, indeed. 
If we return null, potentially the closing of the dialog will be instant and any animation (we don't have currently) will suffer. Do you think it would make a difference if we render the empty dialog? 
@ccanos

Reply:
nah, leave it. You're right I didn't think about animations, good point. I don't think a closed dialog can be heavy at all. Agreed